package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "react-chatlog",
   "version": "0.1.0",
-  "private": true,
+  "private": false,
+  "homepage": "https://coolcud.github.io/react-chatlog/",
```

Comment:
I asked some instructors and it seems like the last `/` might be the problem. Try `"https://coolcud.github.io/react-chatlog"`